optaplanner-workbench-models/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-integration</artifactId>
```

Comment:
@cuijulian This is not good.

[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.optaplanner:optaplanner-workbench-models:pom:7.32.0-SNAPSHOT
[WARNING] 'parent.relativePath' of POM org.optaplanner:optaplanner-workbench-models:[unknown-version] (/home/jenkins/workspace/KIE/master/daily-build/daily-build-pipeline-master/droolsjbpm-integration/optaplanner-workbench-models/pom.xml) points at org.drools:droolsjbpm-integration instead of org.optaplanner:optaplanner-integration, please verify your project structure @ org.optaplanner:optaplanner-workbench-models:[unknown-version], /home/jenkins/workspace/KIE/master/daily-build/daily-build-pipeline-master/droolsjbpm-integration/optaplanner-workbench-models/pom.xml, line 6, column 11